api/src/services/keycloak-service.ts

```diff
@@ -38,22 +49,29 @@ export type KeycloakUser = {
 const defaultLog = getLogger('services/keycloak-service');
 
 /**
- * Service for calling the keycloak admin API.
+ * Service for calling the Keycloak Gold Standard CSS API.
  *
- * Keycloak Documentation (select version and see `Administration REST API` section):
- * - https://www.keycloak.org/documentation-archive.html
+ * API Swagger Doc: https://api.loginproxy.gov.bc.ca/openapi/swagger#/Users/get__environment__basic_business_bceid_users
  *
  * @export
  * @class KeycloakService
  */
 export class KeycloakService {
-  keycloakTokenHost: string;
+  // Used to authenticate with the CSS API using the SIMS API credentials
```

Comment:
+1 for labelling all the keycloak variables 👍

Reply:
Yeah, there are more keycloak env vars now (or soon will be):  Regular user vars, service client vars, css api vars.